examples/word_language_model/README.md

```diff
@@ -0,0 +1,18 @@
+
+Mostly adapted from [here](https://github.com/pytorch/examples/tree/master/word_language_model).
```

Comment:
Maybe a sentence or 2 about what is just re-used, what was added, what inferno contributed.